src/fastgltf.cpp

```diff
@@ -518,7 +518,16 @@ std::string_view fg::URIView::fragment() const noexcept { return _fragment; }
 fs::path fg::URIView::fspath() const {
 	if (!isLocalPath())
 		return {};
+#ifdef _WIN32
+#if __cplusplus >= 202002L
```

Comment:
1. `FASTGLTF_CPP_20` is the more appropriate, platform-independent macro that should be used here.
2. Maybe we should check for [`__cpp_char8_t`](https://en.cppreference.com/w/cpp/feature_test#cpp_char8_t) instead?